spyder/widgets/projects/tests/test_project_explorer.py

```diff
@@ -15,12 +19,38 @@
 from spyder.widgets.projects.explorer import ProjectExplorerTest
 
 @pytest.fixture
-def setup_projects_explorer(qtbot):
+def setup_projects_explorer(qtbot, directory=None):
```

Comment:
Please change this name to `setup_project_explorer`

Reply:
And the right way to achieve what you want is by using a marker to modify the fixture's result. See for example:

https://github.com/spyder-ide/spyder/blob/3.x/spyder/plugins/tests/test_ipythonconsole.py#L75-L79
https://github.com/spyder-ide/spyder/blob/3.x/spyder/plugins/tests/test_ipythonconsole.py#L94-L97
https://github.com/spyder-ide/spyder/blob/3.x/spyder/plugins/tests/test_ipythonconsole.py#L231

Reply:
On a second thought, let's rename the fixture to `project_explorer` (instead of `setup_project_explorer`).